bemserver_ui/static/scripts/modules/views/services/cleanupManage.js

```diff
@@ -46,6 +55,18 @@ class ServiceCleanuManageView {
                 this.#updateServiceState(this.#stateOnRadioElmt.checked);
             }
         });
+        for (let sortRadioElmt of this.#sortRadioElmts) {
+            sortRadioElmt.addEventListener("change", (event) => {
+                event.preventDefault();
+                let sortData = sortRadioElmt.id.split("-");
+                let newSortValue = `${sortData[2].toLowerCase() == "asc" ? "+" : "-"}${sortData[1].toLowerCase()}`;
+                if (this.#sortInputElmt.value != newSortValue) {
+                    this.#sortInputElmt.value = newSortValue;
+                    this.#formFiltersElmt.submit();
+                }
+                console.log("Sort Value : ",newSortValue);
```

Comment:
`console.log` or prints, used for debug purposes, must be remove before submitting code.